lessons/topics/05-Css-paradigms/README.md

```diff
@@ -0,0 +1,22 @@
+# Css Paradigms
+
+At this example we are going to study the different paradigms used to maintain our css code in projects.
+* OOCSS
+* SMACSS
+* BEM
+* Atomic CSS 
+* BEM
+
+## Previous Knowledge
+
+List or links to this repository for previous Knowledge topics eg.:
+
+* Css and how does it work
```

Comment:
Please provide a link to check quickly the docs for CSS and HTML. It would be helpful

https://developer.mozilla.org/en-US/docs/Web/HTML
https://developer.mozilla.org/en-US/docs/Web/CSS

Reply:
You can wrap CSS and HTML with a link using markdown

Reply:
@Gracrys just a little helpful links to add for HTML and CSS. This should be wrapped with the words because the references section is only references/links about the topic .